src/config/settings.py

```diff
@@ -142,7 +142,7 @@
 STATIC_URL = "static/"
 STATIC_ROOT = BASE_DIR / "static"
 
-STATICFILES_DIRS = [BASE_DIR / "static/bootstrap"]
+STATICFILES_DIRS = [BASE_DIR / "static/bootstrap", "src/static/pdfs"]
```

Comment:
src/static/pdfs? lets point it to base dir

Reply:
It doesnt let me do that, throwing The STATICFILES_DIRS setting should not contain the STATIC_ROOT setting.

Reply:
i can do it like this though STATICFILES_DIRS = [BASE_DIR / "static/bootstrap", BASE_DIR/"static/pdfs"], does it make more seance?